Empire/Empire.cpp

```diff
@@ -1458,6 +1498,8 @@ int Empire::StockpileID(ResourceType res) const {
         return m_capital_id;
         break;
     case RE_INDUSTRY:
+        return IMPERIAL_OBJECT_ID;
```

Comment:
Storing an empire's stockpile as an empire meter makes some sense, especially if you're also storing the stockpile limit as a meter... Avoids the need for extra getter/setter functions and effects and valueref stuff, at least in principle... I think there aren't actually relevant valuerefs now to access an empire's meter values, though the EmpireMeterValue condition exists. That said, stockpile amounts should be persistent, so will have a different sort of behaviour from the other current empire meters, which are reset and re-calculated by effects each turn. It also makes sense to keep the stockpile tracking in the relevant resourcepool... By default I'd leave it in the resourcepool.